### DISCUSSION TITLE
Programming other AVR micro controllers

### DISCUSSION THREAD

**Discussion by 0ElectroNoob0** [revoxhere/duino-coin#873] [Q&A]
How do you program other AVR micro controllers other than Arduino Uno, Mega, Esp8266 and Esp32? I'm currently trying to program an Atmega16 wich I can perfectly program with the Arduino Duino coin program but when I start the AVR miner all shares are rejected so I think there must be somthing missing.

**Comment by Doofus4ever** [comment]
Maybe there is no support for it yet i have a Microchip Atmega4809 Xplained pro micro controller has the same chip as the Arduino Everything both have the Atmega4809 but Kolka is not increasing the difficulty but before pools were introduced it used to work so i don`t know
